src/server.rs

```diff
@@ -424,6 +421,8 @@ mod test {
     use store::MemStore;
     use super::*;
 
+    use uuid::Uuid;
```

Comment:
This is an implementation detail of ClientId that the server should not need to know about.